.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
```

Comment:
I think it's best to release on `published` instead of `created`. WDYT?

Reply:
That depends! Type ‘published’ will be triggered if the release is published, no matter it’s draft or not. Type ‘created’ will be triggered when a NON-draft release is created and published. Reference: https://github.community/t/what-is-the-created-type-in-on-release/17184

Reply:
Ah, that makes sense. I assumed that the opposite was the case (from the names only). Thanks for the reference! I agree with you, `created` is the way to go.